releasenotes/notes/deprecate-v1-3cdeb826e55f98cb.yaml

```diff
@@ -0,0 +1,4 @@
+---
+deprecations:
+  - |
+    PrimitivesV1 has been deprecated. Use PrimitivesV2 instead.
```

Comment:
I don't know if people would understand what "PrimitivesV1" are, adding a list of what the deprecated classes are would help.